calico-cloud/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -39,7 +39,10 @@ If the CLI scanner is part of your pipeline, scanning is done before runtime and
 1. On the **Managed Clusters** page, select the cluster from the list, and click **Reinstall**.
 1. Copy the updated installation script command and run it against your cluster.
 
-### Download the scanner
+### Start the scanner
+
+<!--
+If you change the name of this section, open a ticket to update the hardcoded Candu link in the UI. -->
```

Comment:
Specifically, the side-effect is due to changing the section title. 

Anything else in this section can be changed without any concerns.

Reply:
Agree. Thanks.